site/src/pages/release-notes/1.25.0.mdx

```diff
@@ -0,0 +1,143 @@
+---
+date: 2023-08-22
+---
+
+## 🌟 New features
```

Comment:
Notes for the newly added commits:

- You can now easily set trailers to be sent after the data stream using <type://HttpResponse#of(RequestHeaders,Publisher,HttpHeaders)>. #3959 #4727